hack/benchmark/README.md

```diff
@@ -6,9 +6,39 @@ Benchmark 3-member etcd cluster to get its read and write performance.
 
 1. Start 3-member etcd cluster on 3 machines
 2. Update `$leader` and `$servers` in the script
+3. Apply [connection reuse patch](#connection-reuse-patch-for-boom) to [boom code](https://github.com/rakyll/boom) at rev `79153762` and build boom
```

Comment:
If this has to be the specific git SHA 79153762, could we link directly to the URL for that rev, rather than just the rakyll/boom repo?
OR
I realize it is extra work, but I would probably just fork rakyll/boom at that rev into my personal github, apply the patch to my fork's master, and point people there until the patch went upstream.

(Both of these items are just suggestions, and I am OK with this line as it stands if neither suggestion appeals to you.)